fuzz/fuzz_targets/http_api.rs

```diff
@@ -0,0 +1,188 @@
+#![no_main]
+use libfuzzer_sys::fuzz_target;
+use micro_http::Request;
+use once_cell::sync::Lazy;
+use std::os::unix::io::{AsRawFd, RawFd};
+use std::sync::mpsc::{channel, Receiver};
+use std::thread;
+use vmm::api::{http::*, ApiRequest, ApiResponsePayload};
+use vmm::{EpollContext, EpollDispatch};
+use vmm_sys_util::eventfd::EventFd;
+
+static ROUTES: Lazy<Vec<&Box<dyn EndpointHandler + Sync + Send>>> = Lazy::new(|| {
+    let mut keys: Vec<&String> = HTTP_ROUTES.routes.keys().collect();
+    keys.sort();
+    keys.iter()
+        .map(|k| HTTP_ROUTES.routes.get(*k).unwrap())
+        .collect()
+});
```

Comment:
If you were to change `HTTP_ROUTES` from `HashMap` into a `BTreeMap`, you would get the sorted order for free.

And out of curiosity, why do you need these routes to be sorted?

Reply:
That's good advise. Let me give it a try. 

When the values from the `HashMap` is not sorted (say using "HashMap::values()", the same seed/test-case will NOT produce the same results every time. That is bad for reproducing potential crashes/bugs from the fuzzer.